<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1304</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Face Features Based Cursor Controller</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="controlPanelGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Panel de control</string>
    </property>
    <widget class="QPushButton" name="calibrateButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Reestablecer puntos</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="showProcessingCheckbox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Mostrar procesamiento</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="controlMouseCheckbox">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>30</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Controlar el mouse</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="cameraConfigurationGroupBox">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>90</y>
      <width>261</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Configuración de la cámara</string>
    </property>
    <widget class="QSlider" name="brightnessSlider">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="contrastSlider">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="saturationSlider">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="hueSlider">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="brightnessLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Brillo</string>
     </property>
    </widget>
    <widget class="QLabel" name="contrastLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Contraste</string>
     </property>
    </widget>
    <widget class="QLabel" name="saturationLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Saturacion</string>
     </property>
    </widget>
    <widget class="QLabel" name="hueLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Matiz</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="cameraGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>661</width>
      <height>521</height>
     </rect>
    </property>
    <property name="title">
     <string>Cámara</string>
    </property>
    <widget class="CursorControllerWidget" name="cursorControllerWidget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>640</width>
       <height>480</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="graphicsGroupBox">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>90</y>
      <width>341</width>
      <height>521</height>
     </rect>
    </property>
    <property name="title">
     <string>Gráficos</string>
    </property>
    <widget class="Grapher" name="positionGrapher" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>320</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
    <widget class="Grapher" name="velocityGrapher" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>320</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="positionLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Posición de target</string>
     </property>
    </widget>
    <widget class="QLabel" name="velocityLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Velocidad de target</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1304</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="calibrateAction">
   <property name="text">
    <string>Calibrar</string>
   </property>
  </action>
  <action name="showProcessingAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mostrar procesamiento</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CursorControllerWidget</class>
   <extends>QWidget</extends>
   <header>cursorcontrollerwidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Grapher</class>
   <extends>QWidget</extends>
   <header>grapher.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
